iOS/Flows/Conversation/Cells/MessageFooterView.swift

```diff
@@ -37,7 +37,7 @@ class MessageFooterView: BaseView {
         self.readView.pin(.right)
         self.readView.pin(.top)
 
-        self.replySummary.sizeToFit()
```

Comment:
I removed this so that we don't have to do a bunch of complicated layout stuff to figure out how tall and wide the views are.